.github/workflows/build.yml

```
@@ -0,0 +1,33 @@
+name: Scala CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        java: ['adopt@1.8', 'adopt@1.11']
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up sbt and java
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Run tests
+        shell: bash
+        env:
+          ENCRYPTION_PASSWORD: ${{secrets.ENCRYPTION_PASSWORD}}
+        run: |
+          JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8 ./build/build.sh
```

This can be on one line
```suggestion
        run: JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8 ./build/build.sh
```

---

i agree with this. i was just keeping the script what it originally was. i'll make the change